starter-code/src/App.js

```diff
@@ -1,25 +1,36 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import data from './countries.json'
+import CountryDetail from './countrydetail'
+import { Switch, Route } from 'react-router-dom';
+
+import { Link } from 'react-router-dom';
+
 
 class App extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      country : data
+    }
+  }
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        {
+          this.state.country.map( (element, index) => {
+            return (
+              <div className="row" key={index}>
```

Comment:
Both the country list and the <CountryDetail /> component should be inside of the same .row. That is why you are seeing the country info at the bottom of the page.